wiki/Beatmap_Editor/Compose/en.md

```diff
@@ -60,53 +60,55 @@ Hold `Alt` to trigger Distance Snap. The value can range from x0.1 - x2.0.
 
-- Eighth Beats
+- Eighth beats
 - Adds yellow ticks, in addition to the red and blue
 
-**Insert Break Time** button will start/stop "Break Time" in current timestamp and **x/y** is current mouse coordinate.
+<!-- No 1/16 divisor? -->
+
+The **Insert Break Time** button places a ![break](/wiki/Glossary/#break) between two well-spaced objects on the timeline.
 
 ### Centre-Left (Selector)
 
 ![Hit Notes Selector](img/BE_HO.jpg "Hit Notes Selector")
 
 **Your construction toolkit, made for DIY (do it yourself).**
 
-**Please note that in osu!standard's default notes hit sounds works differently in osu!mania**
+**Please note that in osu!standard, default note hitsounds work differently than in osu!mania**
 
-- adding clap on a notes will play 'hitclap' **+ hitnormal** sound on hit while in osu!mania: will only play 'hitclap' sound on hit
+- In osu!standard, adding a clap hit sound on a note will play 'hitclap' **+ hitnormal** sound on hit, while in osu!mania, only 'hitclap' will play.
 
 #### Selector
 
 | Buttons / Header (Keyboard Shortcut) | Usage | Description |
 | ------------------------------------ | ----- | ----------- |
-| Sampleset | Auto, Normal, Soft, Drum | This ignore the timing section's Sampleset setting , and add the selected one... e.g. Timing point with **soft 77%**, While this Sampleset set into **Drum** and have **Clap** on the notes, will play default drum-hitnormal 77% + drum-hitclap 77% on hit |
+| Sampleset | Auto, Normal, Soft, Drum | This will ignore the timing section's Sampleset setting , and adds the selected samplset... e.g. Timing point with **soft 77%**, while this Sampleset set into **Drum** and have **Clap** on the notes, will play default drum-hitnormal 77% + drum-hitclap 77% on hit |
 | Additions | Auto, Normal, Soft, Drum | Additions hitsound means add Clap, Finish, and Whistle with the SELECTED set... e.g. Sampleset **Normal**, Timing point **Soft 77%**, while Additions is set to **Drum** and have **Clap** on the notes, will result of normal-hitnormal 77% + drum-hitclap 77% on hit |
 | Select (`1`) | **Left-click/drag:** Select/Adjust the notes/sliderpoints location. **Right-click:** Remove notes/sliderpoints. **`Ctrl` + Click:** Multiple selection.	| Select and modify existing notes. |
-| Circle (`2`) | **Left/Right-click:** Add/Remove hit circle. | Hit Circle. |
+| Circle (`2`) | **Left/Right-click:** Add/Remove hit circle. | Hit circle. |
 | Slider (`3`) | **Left/Right-click:** Start/End Slider. Add/Remove sliderpoints (white square). **Left-Click x2:** New curve section (red square). | Slider. |
-| Spinner (`4`) | **Left/Right-click:** Start/End Spinner at current timestamp. | Spinner. Make sure Auto can at least score 2,000 Bonus per Spinner (to prevent the infamous spinner spam/ninja spinner). |
+| Spinner (`4`) | **Left/Right-click:** Start/End Spinner at current timestamp. | Spinner. Make sure Auto can at least score 1,000 Bonus per Spinner (to prevent the infamous spinner spam/ninja spinner). |
```

Comment:
can at least into can get at least

Reply:
Change brackets content into (to prevent making spam/ninja spinner) or something [citation needed]